solution/ui/regulations/eregs-vite/src/utilities/api.js

```diff
@@ -416,12 +416,20 @@ const getSupplementalContent = async (
     sString = `${sString}&paginate=true&page_size=${page_size}&page=${page}`
     sString = `${sString}&fr_grouping=${fr_grouping}`
 
-    const urlpath = (window.location.href.match('searchgov')) ? `resources/search?${sString}` : `resources/?${sString}`
-    const response =  await httpApiGet(urlpath)
+    const response =  await httpApiGet(`resources/?${sString}`)
     return response
 }
 
-const getCategories = async () =>  httpApiGet("resources/categories");
+/**
+ * @param {string} [page=1] - page number of paginated results to return.
+ * @param {string} [q=""] - search querystring
+ *
+ * @returns {Promise<{count: number, next: string, previous: string, results: Array<Object>}>} - Promise that contains response object when fulfilled
+ */
+const getSearchGovResources = async ({ page = 1, q = "" }) =>
+    httpApiGet(`resources/search?q=${q}&page=${page}`);
```

Comment:
This is a note, will need to update this line for uriencoding once my current PR is in.

Reply:
This has been updated